spec/models/seed_uri_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'spec_helper'
+
+RSpec.describe SeedUri do
+  describe 'validation' do
+    let(:seed) { described_class.new(uri: uri) }
+    subject { seed.valid? }
+
+    context 'when it is valid' do
+      let(:uri) { 'http://www.slac.stanford.edu/' }
+      it { is_expected.to be true }
+    end
+
+    context 'when it is not valid' do
+      let(:uri) { '"http://www.slac.stanford.edu/"' }
```

Comment:
Would suggest some additional tests of when not valid, including:
* trailing space
* non-ascii characters

Reply:
👍